src/python/src/README.rst

```diff
@@ -6,22 +6,18 @@ Package for GRPC Python.
 Dependencies
 ------------
 
-Ensure that you have installed GRPC core.
-
-On debian linux systems, install from our released deb package:
+Ensure you have installed the gRPC core.  On Mac OS X, install homebrew_. On Linux, install linuxbrew_.
+Run the following command to install gRPC Python.
 
 ::
 
-  $ wget https://github.com/grpc/grpc/releases/download/release-0_5_0/libgrpc_0.5.0_amd64.deb
-  $ wget https://github.com/grpc/grpc/releases/download/release-0_5_0/libgrpc-dev_0.5.0_amd64.deb
-  $ sudo dpkg -i libgrpc_0.5.0_amd64.deb libgrpc-dev_0.5.0_amd64.deb
-
-Otherwise, install from source:
+  $ curl -fsSL https://goo.gl/getgrpc | bash -s python
 
-::
+This will download and run the [gRPC install script][] to install the grpc core, then uses pip to install this package.  It also installs the Protocol Buffers compiler (_protoc_) and the gRPC _protoc_ plugin for python.
```

Comment:
The wording is a bit awkward. How about:

This will download and run [gRPC install script][] to install grpc core. The script then uses pip to install this package. ...